src/components/CardMenus/Tabs.js

```diff
@@ -2,9 +2,10 @@ import React from 'react';
 
 import { Paper, Tab, Tabs } from '@material-ui/core';
 
+import CardMenus from './CardMenus';
+
 const MainTab = () => {
   const [value, setValue] = React.useState(2);
```

Comment:
can we import useState from react library directly? why did you initialize it from 2?